base/remote/R/remote.copy.from.R

```diff
@@ -18,8 +19,8 @@
 #'   host <- list(name='geo.bu.edu', user='kooper', tunnel='/tmp/geo.tunnel')
 #'   remote.copy.from(host, '/tmp/kooper', '/tmp/geo.tmp', delete=TRUE)
 #' }
-remote.copy.from <- function(host, src, dst, delete = FALSE, stderr = FALSE) {
-  args <- c("-az", "-q")
+remote.copy.from <- function(host, src, dst, options = c(), delete = FALSE, stderr = FALSE) {
```

Comment:
I personally think `NULL` is a bit cleaner, easier to test against, and is more consistent with default arguments elsewhere in PEcAn.

```suggestion
remote.copy.from <- function(host, src, dst, options = NULL, delete = FALSE, stderr = FALSE) {
```

Reply:
(Note that you can apply both of these changes as a single commit if you go to the "Files" tab and use the "Add suggestion to batch" feature. Very handy!)

Reply:
oh rad, I see what you mean, I'll do that next time!